src/main/java/org/generationcp/middleware/dao/AttributeDAO.java

```diff
@@ -26,6 +26,9 @@
 import java.util.ArrayList;
 import java.util.List;
 
+import java.util.ArrayList;
```

Comment:
@vmaletta 
Please verify these 2 new imports, seems to be duplicated.

Reply:
Done